### ISSUE TITLE
k2cli force option to proceed with depreciated schema

### ISSUE THREAD

**Issue by davidewatson** [samsung-cnct/k2cli#66]
Changed to:

Because we expect warnings to be ignored (or missed due to the sheer volume of ansible output), when a stanza is depreciated k2/k2cli should not proceed. Instead, k2cli should have a force option which is passed through to k2. This option will force the admin to acknowledge the depreciation and allow the admin to continue using depreciated schema. If the option is not given, a textual explanation of the changes should be printed, and the operation should stop.

Was:

# Task

Per #262, k2 will print warnings whenever a feature is depreciated. k2cli is a user interface for k2 which uses a spinner to indicate progress while hiding k2s intentionally verbose output. This ticket is to print warnings whenever appropriate.

**Comment by davidewatson** [comment]
As an aside, we might want to stop using a spinner altogether in favor of something more user friendly. See [this](https://uxplanet.org/progress-indicators-in-mobile-ux-design-a141e22f3ea0#.1zlepb7ju) article for ideas. Any work on this might be a separate ticket, but keep this in mind during implementation.

**Comment by coffeepac** [comment]
blocked by #258
